packages/flame/lib/src/game/mixins/game.dart

```diff
@@ -157,17 +157,8 @@ mixin Game on Loadable implements Projector {
     return _gameRenderBox!.localToGlobal(point.toOffset()).toVector2();
   }
 
-  @override
-  Vector2 unprojectVector(Vector2 vector) => vector;
-
-  @override
-  Vector2 projectVector(Vector2 vector) => vector;
-
-  @override
-  Vector2 unscaleVector(Vector2 vector) => vector;
-
-  @override
-  Vector2 scaleVector(Vector2 vector) => vector;
+  Projector projector() => IdentityProjector();
```

Comment:
Should we make these attributes instead of methods, so we avoid instantiating this more than necessary?

Reply:
Good point, forgot you could override attributes on dart. let me see how it looks